packages/storage/docs/stories/UpdateUserPhotoModal.story.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { RainbowFirebaseApp } from '@rainbow-modules/app';
+import {
+    WhenAuthenticated,
+    WhenNoAuthenticated,
+    EmailPasswordSignInForm,
+} from '@rainbow-modules/auth';
+import { Avatar, Button } from 'react-rainbow-components';
+import { getAuth, onAuthStateChanged } from 'firebase/auth';
+import { Camera } from '@rainbow-modules/icons';
+import styled from 'styled-components';
+import app from '../../../../firebase';
+import UpdateUserPhotoModal from '../../src/components/UpdateUserPhotoModal';
+
+const StyledButton = styled(Button)`
+    border-radius: 50%;
+    width: 30px;
+    height: 30px;
+    display: flex;
+    align-content: center;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    margin: -25px 0 0 40px;
+`;
+
+const StyledAvatar = styled(Avatar)`
+    width: 70px;
+    height: 70px;
+`;
+
+export const BasicUpdateUserPhotoModal = () => {
+    const [photoURL, setPhotoURL] = useState();
+    const [isOpen, setIsOpen] = useState(false);
+    const [initials, setInitials] = useState();
+
+    const getInitials = (name) => {
+        if (name) {
+            const names = name.split(' ');
+            const initials =
+                names[0].substring(0, 1).toUpperCase() + names[1].substring(0, 1).toUpperCase();
+            return initials;
+        }
+        return '';
+    };
```

Comment:
since this function does not depend on any prop or state it can be declared outside